modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -79,6 +82,14 @@ public BrowserWebDriverContainer() {
                 .withStrategy(new HostPortWaitStrategy())
                 .withStartupTimeout(Duration.of(15, SECONDS));
 
+        try {
```

Comment:
IMO we should move it to `configure` and create the dir only if recording is enabled

Reply:
Moved the code.